Src/CZICmd/CMakeLists.txt

```
@@ -26,23 +26,44 @@ IF(UNIX)
  ENDIF()
 ENDIF(UNIX)
 
-find_package(RapidJSON QUIET)
-IF (NOT RapidJSON_FOUND)
-    message(FATAL_ERROR [=[
-    RapidJSON library was not found, which is required for building. Consider installing
-    like 'sudo apt-get install rapidjson-dev'.
-    ]=])
-ENDIF()
-
 include(FetchContent)
 
+if (LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_RAPIDJSON)
+    find_package(RapidJSON QUIET)
+    IF (NOT RapidJSON_FOUND)
+        message(FATAL_ERROR [=[
+        RapidJSON library was not found, which is required for building. Consider installing
+        like 'sudo apt-get install rapidjson-dev'. Alternatively, consider setting the option
+        LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_RAPIDJSON to OFF in order to download and build RapidJSON
+        automatically as part of the build process.
+        ]=])
+    ENDIF()
```

```suggestion
    endif()
```
Lowercase for consistency